test/Sentry.Tests/GlobalSessionManagerTests.cs

```diff
@@ -291,13 +293,21 @@ public void TryGetPersistentInstallationId_SessionStarted_ReturnsLastSession()
         // Arrange
         var sut = _fixture.GetSut();
 
+        var timeOffset = DateTimeOffset.Now;
+        _fixture.Clock.GetUtcNow().Returns((_) =>
+        {
+            timeOffset = timeOffset.AddSeconds(1);
+            return timeOffset;
+        });
+
         var sessionUpdate = sut.StartSession();
 
         // Act
         var persistedSessionUpdate = sut.TryRecoverPersistedSession();
 
         // Assert
         sessionUpdate.Should().NotBeNull();
+        persistedSessionUpdate!.EndStatus.Should().Be(SessionEndStatus.Abnormal);
```

Comment:
perhaps move the `!` up to `var persistedSessionUpdate = sut.TryRecoverPersistedSession()!;`. it better represents the intent?